packages/vite/src/node/utils.ts

```diff
@@ -1070,7 +1070,7 @@ export const requireResolveFromRootWithFallback = (
 }
 
 export function emptyCssComments(raw: string): string {
-  return raw.replace(multilineCommentsRE, (s) => ' '.repeat(s.length))
+  return raw.replace(multilineCommentsRE, (s) => blankReplacer(s))
```

Comment:
```suggestion
  return raw.replace(multilineCommentsRE, blankReplacer)
```

I think we can pass this directly too.

Reply:
Thanks for the review! I've rebased the branch and pushed the change.